frontend/src/old-pages/Clusters/Properties.tsx

```diff
@@ -100,36 +100,25 @@ export default function ClusterProperties() {
                     triggerType="custom"
                     content={
                       <StatusIndicator type="success">
-                        ARN copied
+                        SSH Command copied
                       </StatusIndicator>
                     }
                   >
                     <Button
                       variant="inline-icon"
                       iconName="copy"
-                      ariaLabel="Copy ARN"
+                      ariaLabel="Copy SSH Command"
                       onClick={() => {
                         navigator.clipboard.writeText(
-                          cluster.cloudformationStackArn,
+                          `ssh ${clusterDefaultUser(cluster)}@${headNode.publicIpAddress}`
                         )
                       }}
                     >
                       copy
                     </Button>
                   </Popover>
                 </Box>
-                <Link
-                  external
-                  href={`${consoleDomain(
-                    cluster.region,
-                  )}/cloudformation/home?region=${
-                    cluster.region
-                  }#/stacks/events?filteringStatus=active&filteringText=&viewNested=true&hideStacks=false&stackId=${
-                    cluster.cloudformationStackArn
-                  }`}
-                >
-                  {cluster.cloudformationStackArn}
-                </Link>
+                ssh {clusterDefaultUser(cluster)}@{headNode.publicIpAddress}
```

Comment:
question: I'm lacking context on why this feature is needed, especially when there's already an integration with SSM. 
Some questions that can frame the problem better:
- Is this needed to connect to the head node for troubleshooting? 
- Why a user should use SSH over SSM?
- If a user has to connect to the head node, they are going to need a key-pair or some kind of authentication mechanism, so the command, as it is right now, is not usable in a production environment. What's the value of the it?

Reply:
So the idea here is to make it easier for users to use their native SSH client to connect. Yes they will need a keypair, however if they've added their keypair as the default key (either by naming it `id_rsa` or setting that in their ssh config) they can connect using that command. I'd love to include an option to download the keypair also (or upload their pub key in PCM) but that's a much bigger feature.

There's a number of reasons users would want to connect via SSH instead of SSM.
1. They don't have console access. 
2. The cluster isn't setup with SSM access
3. They have a multi-user environment setup and need to authenticate with SSH
4. They want to use their local CLI instead of the browser
5. They want to copy files (using scp or sftp)